clients/client-python/tests/integration/integration_test_env.py

```diff
@@ -45,43 +46,43 @@ class IntegrationTestEnv(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         if os.environ.get('START_EXTERNAL_GRAVITINO') is not None:
-            """Maybe Gravitino server already startup by Gradle test command or developer manual startup."""
+            # Maybe Gravitino server already startup by Gradle test command or developer manual startup.
```

Comment:
Is `#` the recommend comment format for Python?

Reply:
It seems that pylint think this is a long string, not a comment. Maybe try to add new line in this?

Reply:
https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/pointless-string-statement.html

Maybe we need to determine whether to disable this rule to allow long string comment.

Reply:
I think the comment should use `#` and the docstring use `"""`. The case here is more like a comment.